deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,572 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.1.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.1.0
+    createdAt: "2020-02-19T13:14:00Z"
```

Comment:
Is there a script for creating this? if so, where?

Reply:
no script, We'll update it to the date when we publish the operator to OperatorHub.

Reply:
Is this detail included in some ticket/checklist? I think most of the team are not aware of this.

Reply:
It is done in the ticket "publish operator to OperatorHub" in v1.0.0. But yes, no details.

Reply:
We cannot leave it like this with no details anywhere. This (and other details) can easily be missed.

Reply:
Thanks for covering this in ticket CSI-1205